web/src/project/ProjectSummary.tsx

```diff
@@ -441,8 +447,14 @@ export const ProjectSummary = () => {
             return <></>;
         }
         return Object.entries(summary.seqr_links).map(([key, value]) => (
```

Comment:
Can you add a title above the seqr buttons?

Reply:
Donezo